documentation/library_guide/oneDPL_2022.0_changes.rst

```diff
@@ -0,0 +1,40 @@
+oneDPL 2022.0 Changes
+#####################
+
+This page lists breaking changes in oneDPL 2022.0 from oneDPL 2021.
```

Comment:
This page list the breaking changes that occurred between the 2021.0 and 2022.0 releases.

Reply:
Applied the suggestion.